Sources/JavaScriptKit/BasicObjects/JSTypedArray.swift

```diff
@@ -106,12 +110,16 @@ public class JSTypedArray<Element>: JSBridgedClass, ExpressibleByArrayLiteral wh
         @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
         public func withUnsafeBytesAsync<R>(_ body: (UnsafeBufferPointer<Element>) async throws -> R) async rethrows -> R {
             let bytesLength = lengthInBytes
-            let rawBuffer = malloc(bytesLength)!
-            defer { free(rawBuffer) }
-            _load_typed_array(jsObject.id, rawBuffer.assumingMemoryBound(to: UInt8.self))
-            let length = lengthInBytes / MemoryLayout<Element>.size
```

Comment:
@yonihemi Do you remember whether it's intentional or not to read `lengthInBytes` twice?